ironfish-cli/src/commands/chain/import.ts

```diff
@@ -55,20 +57,15 @@ export default class ImportSnapshot extends IronfishCommand {
         this.exit(1)
       }
 
-      const client = await this.sdk.connectRpc()
+      const client = await this.sdk.connectRpc(true)
       const status = await client.getChainInfo()
 
       const manifest = (await axios.get<SnapshotManifest>(`${bucketUrl}/manifest.json`)).data
+      const fileSize = FileUtils.formatFileSize(manifest.file_size)
 
       if (!flags.confirm) {
         this.log(
-          `This snapshot (${
-            manifest.file_name
-          }) contains the Iron Fish blockchain up to block ${
-            manifest.block_height
-          }. The size of the latest snapshot file is ${FileUtils.formatFileSize(
-            manifest.file_size,
-          )}`,
+          `This snapshot (${manifest.file_name}) contains the Iron Fish blockchain up to block ${manifest.block_height}. The size of the latest snapshot file is ${fileSize}`,
```

Comment:
I know it was this way before, so this is just a note for more work, we should rename height -> sequence in the manifest and the logs. We call it sequence in our code base and our logs. We can do that in another PR.

Reply:
Updated to sequence